CHANGES.rst

```diff
@@ -1195,6 +1195,9 @@ astropy.units
 - For ``Quantity.to_string()``, ensure that the precision argument is also
   used when the format is not latex. [#11145]
 
+- For CDS units, recognize ``-`` as indicating dimensionless and ``[-]``
```

Comment:
Maybe say `For CDS units and tables, ...` to make that connection to the common use case driving this.

Reply:
OK, done